api/base/utils.py

```diff
@@ -153,21 +153,25 @@ def waterbutler_url_for(request_type, provider, path, node_id, token, obj_args=N
     url.args.update(query)
     return url.url
 
-def default_node_list_queryset():
-    return Node.objects.filter(is_deleted=False)
+def check_model_cls(model_cls):
```

Comment:
Minor: The name of this function doesn't really suggest what it's for--keep in mind that this is in `api.base.utils`, which contains general-purpose helpers. You could rename it to `is_node_or_registration_cls(model_cls)`, but it might be simpler to just get rid of the extra layer of indirection and do `assert model_cls in {Node, Registration}` where necessary.